internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -168,15 +183,31 @@ func (db *gormdb) Setup(ctx context.Context) error {
 func (db *gormdb) Teardown(ctx context.Context) error {
 	isSqlite := db.cfg.Type == "sqlite"
 	if isSqlite {
-		db.db.Exec("PRAGMA foreign_keys = OFF")
+		foreignKeys(db, false)
 	}
 	if err := db.db.WithContext(ctx).Migrator().DropTable(scheme.Tables...); err != nil {
 		return fmt.Errorf("droptable: %w", err)
 	}
 	if isSqlite {
-		db.db.Exec("PRAGMA foreign_keys = ON")
+		foreignKeys(db, true)
+	}
+
+	return nil
+}
+
+func (gormdb *gormdb) Cleanup(ctx context.Context) error {
+	foreignKeys(gormdb, false) // disable foreign keys
+
+	db := gormdb.db.WithContext(ctx).Unscoped().Session(&gorm.Session{AllowGlobalUpdate: true})
```

Comment:
Isn't it's easier to create a `fmt.Sprintf("DELETE FROM %s", mode.name)` an run it?

Reply:
emm.. maybe. But I definitely prefer to use gorm and not raw SQL.
Just to be sure that I understand you right - could you plz provide your version of this function